src/getCss.js

```diff
@@ -0,0 +1,24 @@
+// Manifest of rendered module ids
+const renderedModules = [];
+
+/**
+ * Get CSS content from a module as a string
+ */
+function getCss(styles) {
+  const stylesToRender = styles.filter((module) => {
+    const [moduleId] = module;
+
+    // Check if we've already rendered a module with this ID
+    if (!renderedModules.includes(moduleId)) {
```

Comment:
You should consider making `renderedModules` an object or Map, so that you can do `O(1)` lookups instead of `O(n)`, basically it would be more performant.